packages/components/src/local/organisms/planning-channel/index.tsx

```diff
@@ -531,17 +531,21 @@ export const PlanningChannel: React.FC<PropTypes> = ({
     let localBounds: L.LatLngBounds | undefined
     plans.forEach((plan) => {
       const geom = plan.geometry.geometry as any
-      if (geom.coordinates) {
-        const coords = geom.coordinates as Array<[number, number]>
-        coords.forEach((val: [number, number]) => {
-          const pos = L.latLng(val[0], val[1])
-          localBounds = localBounds === undefined ? L.latLngBounds(pos, pos) : localBounds.extend(pos)
-        })
-      } else if (geom.coordinate) {
-        const val = geom.coordinate as [number, number]
-        const pos = L.latLng(val[0], val[1])
-        localBounds = localBounds === undefined ? L.latLngBounds(pos, pos) : localBounds.extend(pos)
-      }
+      // currently I saw the geom.coordinates has the same format with geom.coordinate, so I guess there is something wrong with the data
```

Comment:
note: I _think_ we receive `coordinate` if there is only one item, e.g. if it's a point rather than a line/polygon

Reply:
hmm, can you help to re-check it 🤔 , I think the `geom.coordinates` handles data for all types: `Array of [number, number]` and `[number, number]`